index.js

```diff
@@ -1,13 +1,14 @@
 var relative = require('relative-date')
 var pretty = require('prettier-bytes')
 var path = require('path')
-var data = require('render-data')
 var yo = require('yo-yo')
+var data = module.parent ? null : require('render-data')
 
 module.exports = Tree
 
 function Tree (root, entries, onclick) {
   if (!(this instanceof Tree)) return new Tree(root, entries, onclick)
+  if (module.parent) onclick = function () {}
```

Comment:
additional comment: if we decide to go with `process.browser` and it is only true when bundled via browserify, we are kind of locking other developers into using browserify, no? yo-fs doesn't really have any opinion re: bundlers as it stands right now, which is probably optimal. still leaning towards `!window` if that's the case.